reto 2 terminado

MiguelMoraissaimai opened the pull request:
esta en entregas, en java bajo el nombre "piedraPapelTijera.java"

MiguelMoraissaimai committed:
paso dos

MiguelMoraissaimai committed:
paso tres

MiguelMoraissaimai committed:
ultimo commit

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
cambios desde web

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

Commit:
java primer commit

Commit:
ultimo commit de arreglo

MiguelMoraissaimai committed:
si

Commit:
commit tras hacer pruebas de texto e integers

Commit:
calculo de nota hecho

Commit:
division añadida entre ejerecicio inicial de prueba de int, double, etc y el ejercicio de las notas

Commit:
ejercicio terminado y arhchivos finalizados

Commit:
arreglo por olvido de archivo eliminado

Commit:
arreglo por olvido de archivo eliminado... otra vez

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

Commit:
mates added

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
generador de numeros aleatorios

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

Commit:
piedraPapelTijera añadido

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
arreglos a piedra papel tijera

MiguelMoraissaimai committed:
reto 2 terminado, puesto en entregas como 'piedraPapelTijera.java'

mmasias commented:
> esta en entregas, en java bajo el nombre "piedraPapelTijera.java"

Pues si está así, el nombre estaría mal, porque tiene que empezar con mayúsculas. Lo paso viendo ahora mismo!

mmasias commented on entregas/moraisMiguel/java/piedraPapelTijera.java:
```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+class piedraPapelTijera{
```
Efectivamente. El nombre debe ser PiedraPapelTijera (y el archivo, PiedraPapelTijera.java). Los puristas dirán que son Tijeras, pero bueno...

mmasias commented on entregas/moraisMiguel/java/piedraPapelTijera.java:
```diff
@@ -0,0 +1,38 @@
+
+        final int MAXIMO = 4;
+        final int MINIMO = 1;
+
+
```
Si usas espacios para separar, de acuerdo, pero que sea uno solo.

mmasias commented on entregas/moraisMiguel/java/piedraPapelTijera.java:
```diff
@@ -0,0 +1,38 @@
+        String descriptor;
+        String resultado = eleccion == eleccionUsuario ? EMPATE
+        : (eleccion == 1 && eleccionUsuario == 2) ||(eleccion == 2 && eleccionUsuario == 3) || (eleccion == 3 && eleccionUsuario == 1) ? GANASTE
+        : (eleccion == 1 && eleccionUsuario == 3) ||(eleccion == 2 && eleccionUsuario == 1) ||(eleccion == 3 && eleccionUsuario == 2) ? PERDISTE
```
En realidad, si no han empatado ni ganado, no queda más que perder, por lo que no hacen falta las validaciones.

Commit:
añadido juegos for

Commit:
added caracol

Commit:
updated caracol

Commit:
akjdnas

MiguelMoraissaimai committed:
derrota

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

Commit:
marco added

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

Commit:
Carrear

Commit:
batalla

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

mmasias committed:
Update README.md

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

mmasias committed:
Segundo intento

mmasias committed:
3rd

mmasias committed:
Enlaces

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

MiguelMoraissaimai committed:
intento de juego de adivinar el numero (me complique demasiado)

MiguelMoraissaimai committed:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
matrices

MiguelMoraissaimai committed:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
mas practica de matrices

MiguelMoraissaimai committed:
practicas

MiguelMoraissaimai committed:
juego del topo

MiguelMoraissaimai committed:
alquimia comienzo

MiguelMoraissaimai committed:
viajemarco

MiguelMoraissaimai committed:
string equality
